templates/model_package_group/sdk_read_one_post_set_output.go.tpl

```diff
@@ -0,0 +1,11 @@
+    if isModelPackageGroupPending(&resource{ko}) {
+        msg := "ModelPackageGroup is currently pending"
+        setSyncedCondition(&resource{ko}, corev1.ConditionFalse, &msg, nil)
+        return &resource{ko}, requeueWaitWhilePending
+    }
+    if isModelPackageGroupInProgress(&resource{ko}) {
+        msg := "ModelPackageGroup is currently in progress"
+        setSyncedCondition(&resource{ko}, corev1.ConditionFalse, &msg, nil)
+        return &resource{ko}, requeueWaitWhileInProgress
+    }
+    ModelPackageGroupCustomSetOutput(&resource{ko})
```

Comment:
duplicate logic between line 1-10 and this line.

I feel this is cleaner, lets use this style. Its in sync with other resources and less duplicated code
nit: rename to customSetOutput